site/layouts/shortcodes/placeholder.html

```diff
@@ -42,7 +42,7 @@
     <rect width="100%" height="100%" fill="{{ $background }}"/>
     {{- if $show_text }}<text x="50%" y="50%" fill="{{ $color }}" dy=".3em">{{ $text }}</text>{{ end -}}
     {{- if $show_icon }}
-    <svg x="30%" y="30%" width="40%" height="40%" viewBox="0 0 24 24" fill="#00000088">
+    <svg x="30%" y="30%" width="40%" height="40%" viewBox="0 0 24 24" fill="#{{ if (or (eq $background "#000") (eq $background "#333")) }}ffffff30{{ else }}00000014{{ end }}">
```

Comment:
Not 100% sure but I'd say there's no placeholder shortcode called with a background set to `#333`.
Doesn't seem to be `$default_background` value since `#333` is not in `grays.yml`.

Reply:
Yeah it was a defensive value with the gray value we have but I could remove it if you prefer

Reply:
Not a strong opinion on my side. Just thinking about the future when we'll try to find what corresponds to this value. I'm not sure but I don't think it would work with `#333333` or stuff in this spirit.
If you have a strong opinion regarding the defensive approach, we can keep it and you can resolve this comment.